untangle-vue-ui/source/src/components/settings/system/Settings.vue

```diff
@@ -44,13 +47,27 @@
         cb(response.success)
       },
 
+      /* Handler to sync time via ntp */
+      async onForceTimeSync() {
+        this.$store.commit('SET_LOADER', true)
+        const response = await this.$store
+          .dispatch('settings/doForceTimeSync')
+          .finally(() => this.$store.commit('SET_LOADER', false))
+        if (response.success && !response.message) {
+          this.$vuntangle.toast.add(this.$vuntangle.$t('force_time_sync_success'))
+        } else {
+          this.$vuntangle.toast.add(this.$vuntangle.$t('force_time_sync_failed'), 'error')
```

Comment:
There was a request on our side to just not show success toasts any longer.
We show only error toasts if it's the case, otherwise the assumption is that the api call was successful. I think you should do the same in the context of NGFW. 

Just do not show any longer success toasts.